matscipy/fracture_mechanics/crack.py

```diff
@@ -1686,39 +2036,38 @@ def strain_err(self,cutoff,seperate_surface=False):
 
         dv : norm strain error for each atom in r.
         """
-        
+
         '''
-        #want to get all neighbours in region I
+        # want to get all neighbours in region I
         I, J = neighbour_list('ij', self.atoms[self.regionI], cutoff)
-        #print(I)
-        #print(J)
+        # print(I)
+        # print(J)
         v = self.u
         dv = np.linalg.norm(v[I, :] - v[J, :], axis=1)
         r = self.r[self.regionI][I]
-        mask = r<(self.rI-cutoff)
+        mask = r < (self.rI-cutoff)
         if seperate_surface:
             mid_cell_x = self.cryst.cell.diagonal()[0] / 2.0
             mid_cell_y = self.cryst.cell.diagonal()[1] / 2.0
             I_positions = self.cryst.get_positions()[I]
-            x_criteria = ((I_positions[:,0] - mid_cell_x) < 0.5)
-            y_criteria = ((I_positions[:,1] - mid_cell_y) < ((1.5 * self.cutoff)))&\
-            ((I_positions[:,1] - mid_cell_y) > (-1.5 * self.cutoff))
-            surface_mask_full = np.logical_and(x_criteria,y_criteria)
+            x_criteria = ((I_positions[:, 0] - mid_cell_x) < 0.5)
+            y_criteria = ((I_positions[:, 1] - mid_cell_y) < ((1.5 * self.cutoff))) &\
+                ((I_positions[:, 1] - mid_cell_y) > (-1.5 * self.cutoff))
+            surface_mask_full = np.logical_and(x_criteria, y_criteria)
             bulk_mask_full = np.logical_not(surface_mask_full)
-            surface_mask = np.logical_and(surface_mask_full,mask)
-            bulk_mask = np.logical_and(bulk_mask_full,mask)
-            return r[surface_mask],dv[surface_mask],r[bulk_mask],dv[bulk_mask]
+            surface_mask = np.logical_and(surface_mask_full, mask)
+            bulk_mask = np.logical_and(bulk_mask_full, mask)
+            return r[surface_mask], dv[surface_mask], r[bulk_mask], dv[bulk_mask]
         else:
-            return r[mask],dv[mask]
-
-        
-    def convergence_line_plot(self,num=0):
-        def gen_mask(r,rval,dx,dr,cx,cy):
-            #for a line going vertically up
-            #so r is essentiall y here
-            rcriteria = (r > rval)&(r<(rval+dr))
-            xcriteria = (x-cx > 0)&(x-cx<dx)
-            return np.logical_and(rcriteria,xcriteria)
+            return r[mask], dv[mask]
+
+    def convergence_line_plot(self, num=0):
```

Comment:
move to scripts?

Reply:
I think maybe keep them here for now, then potentially move them if any more plotting scripts become available in future? I'll make a comment reminder